src/pages/dar_application/DataAccessRequest.js

```diff
@@ -1,22 +1,204 @@
-import { useState, useEffect } from 'react';
+import { useState, useEffect, useRef } from 'react';
 import { a, br, div, fieldset, h, h3, input, label, span, textarea } from 'react-hyperscript-helpers';
 import isNil from 'lodash/fp/isNil';
 import isEmpty from 'lodash/fp/isEmpty';
+import forEach from 'lodash/fp/forEach';
+import includes from 'lodash/fp/includes';
+import cloneDeep from 'lodash/fp/cloneDeep';
+import isEqual from 'lodash/fp/isEqual';
+import every from 'lodash/fp/every';
```

Comment:
Side note, I've come to the realization that this is a better pattern than * imports, similar to what we do on the java side.